src/fairseq2/models/wav2vec2/asr/_factory.py

```diff
@@ -68,6 +68,7 @@ def create_masker(self) -> Wav2Vec2Masker:
         config = self._config
 
         return StandardWav2Vec2Masker(
+            config.mask_codebase,
```

Comment:
to rm

Reply:
Masking is not configurable anymore due to a refactor. Changes here can be disregarded.